### PR TITLE
MAT-7228 ensure test case population values have id and criteria refe…

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
…rence for cv or ratio observations

## MADiE PR Template

Jira Ticket: [MAT-7228](https://jira.cms.gov/browse/MAT-7228)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e. Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/models/validators/TestCasePopulationValueValidator.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                                    .equalsIgnoreCase(value.getName().name())
+                                || PopulationType.NUMERATOR_OBSERVATION
+                                    .name()
+                                    .equalsIgnoreCase(value.getName().name()))
+                            && value.getExpected() != null
+                            && StringUtils.hasText(value.getExpected().toString())
+                            && (!StringUtils.hasText(value.getCriteriaReference())
+                                || !StringUtils.hasText(value.getId())))
+                .toList();
+        isValid = CollectionUtils.isEmpty(denomAndNumerObservations);
```
Comment (RohitKandimalla) [inline src/main/java/gov/cms/madie/models/validators/TestCasePopulationValueValidator.java:62]: Not a big deal, but looks like we could just return the output of `CollectionUtils.isEmpty` and doesn't have to worry about updating isValid and breaking the flow.

Comment (RohitKandimalla) [reply src/main/java/gov/cms/madie/models/validators/TestCasePopulationValueValidator.java:62]: default could just return true;

**Review by RohitKandimalla — APPROVED** [review]

File: src/main/java/gov/cms/madie/models/validators/TestCasePopulationValueValidator.java (hunk 1 above)
Comment (sb-cecilialiu) [reply src/main/java/gov/cms/madie/models/validators/TestCasePopulationValueValidator.java:62]: @RohitKandimalla Thanks so much Rohit! I tend to use one variable and do one return instead of multiple returns in a block. I guess you are right not to use/update isValid as it's less code.
